CONTRIBUTING.md

```diff
@@ -1,95 +1,37 @@
 
-# A. Writing a New Pattern
+3. **Contribute a new pattern.** A key contribution you can make is to document an InnerSource best practice you experienced or know about. We defined different maturity levels for the contents in this repository: First time contributors should aim for maturity levels `1: initial` (for unstructured experience reports or what we call donuts - patterns with missing pieces) and `2: structured` (for best practices adhering to the patterns format and following basic writing style conventions). Our [Contributor Handbook](meta/contributor-handbook.md) discusses the maturity levels in more detail and explains how you can contribute your first pattern.
 
-The below steps can be used to create a new pattern. The use-case here is that you have an idea or problem in your head and can confidently fill out the barest of pattern fields (Solution doesn't need to be known). If you are unsure your idea is ready for this, [discuss it in an issue first](#b-discussing-early-ideas-in-issues).
+4. **Be a patterns advocate.** The patterns will become better and cover more topics the more people contribute their perspective and experience. A colleague told you about a novel InnerSource best practice? Please invite her/him to contribute to this repository!
 
-The simplest way to create a pattern is with your browser (see below).  
-[Like the command-line better? See here for alternate instructions.](/meta/technical-git-howto.md)
 
-<img alt="Creating a new pattern" src="/assets/img/write-new-pattern.png" width="70%">
+## How to get help?
 
-1. Login to GitHub & inside the [patterns web repo](https://github.com/InnerSourceCommons/InnerSourcePatterns), click on the 'Create new file' button
-2. Name the file like this example: "project-management-time-pressures.md"
-3. Use the [pattern template](https://raw.githubusercontent.com/InnerSourceCommons/InnerSourcePatterns/master/meta/pattern-template.md) to create your new [markdown](/meta/markdown-info.md) file with the description of your fledgling pattern; it does not need to be complete, as you can add to it later
-4. Enter a commit message
-   * If you are asked to 'Commit directly' vs 'Create a new branch', see [branching details](#use-branches-when-creating-new-patterns) below.
-5. Propose this new file and then also create a Pull Request (PR)
+We are happy to support you in contributing to the InnerSource patterns. You can reach out to us in the following ways:
```

Comment:
Are you also open to create an issue for help? Good discussion in an issue helps to show what's missing in current document for self guiding. I love how @spier was able to pull the discussion in my #150 and turn that into a document for the mindmap.

Reply:
Creating an issue is an important way of getting involved. I made that a little more explicit in the list above.

If we feel that it is not explicit enough, we could adapt the contributing.md file later on with little pain :)